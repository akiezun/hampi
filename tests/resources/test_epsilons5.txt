S = B B "s" ;
B = | "b" ;