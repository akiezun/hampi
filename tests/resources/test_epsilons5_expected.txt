B = "b"  ;
S = "s"  | B "s"  | B B "s"  ;